base/interactiveutil.jl

```diff
@@ -64,7 +64,27 @@ function edit(path::AbstractString, line::Integer=0)
     nothing
 end
 
-edit(f)          = edit(functionloc(f)...)
+function edit(f)
+    ms = methods(f)
```

Comment:
I missed that `methods` "works" (returns 0 methods) when passed anything other than a `Function`.